package.json

```diff
@@ -606,7 +606,7 @@
         },
         {
           "command": "vscode-objectscript.modifyWsFolder",
-          "when": "vscode-objectscript.connectActive && resourceScheme =~ /^isfs(-readonly)?$/ && explorerResourceIsRoot && !listMultiSelection",
```

Comment:
Removing the `vscode-objectscript.connectActive` check is fine, but why remove the `resourceScheme` one?

Reply:
Because `resourceScheme` is undefined if you haven't opened a file yet. And in any case the file you've got open might not come from the root whose settings you're trying to alter.

Reply:
That sounds really strange to me. I just opened my `.code-workspace` file with two `isfs` folders. No files have been opened besides the "welcome" webview that opens automatically. The `Modify server-side workspace folder...` context menu option is present. Are those not valid reproduction steps?

Reply:
Try starting with no workspace, then add a folder from Server Manager.

Reply:
Ok, now I can see it. This seems like a bug in VS Code though. The `resource*` context keys should be set to the Uri of the explorer tree item when you open the context menu. That works for saved workspaces upon open, but doesn't for untitled workspaces unless you open the folder that was just added.

Reply:
It seems that the `resource*` context keys for the root folder are missing when it's a single-root workspace but are present when it's multi-root.

Reply:
I plan to report it. Meanwhile I've tweaked my change.